pkg/email/templates/aggregated_daily_digest_body.html

```diff
@@ -24,7 +24,6 @@ <h3>{{$audit.ModelName}}</h3>
 
 <br />
 
-<!-- <a href="{{$.ClientAddress}}/models/{{$audit.ModelPlanID}}/collaboration-area/task-list"> View this Model Plan in MINT </a> -->
```

Comment:
Addresses [MINT-3543](https://jiraent.cms.gov/browse/MINT-3543)
this is the only email file that references `task-list`.